src/components/TransactionItemComponent/DataCeils/MerchantCell.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import type TransactionDataCellProps from './TransactionDataCellProps';
+
+function MerchantCell({transactionItem, showTooltip}: TransactionDataCellProps) {
+    const styles = useThemeStyles();
+
+    const merchantToDisplay = !transactionItem?.merchant || transactionItem?.merchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT ? '' : transactionItem.merchant;
+    return (
+        <TextWithTooltip
+            shouldShowTooltip={showTooltip}
+            text={merchantToDisplay ?? ''}
```

Comment:
```suggestion
            text={merchantToDisplay}
```

merchantToDisplay is a string type, we don't need to add a fallback value